.github/test-matrix.json

```diff
@@ -0,0 +1,110 @@
+{
+  "include": [
+    {
+      "modules": [
+        "core/trino-main"
+      ]
+    },
+    {
+      "modules": [
+        "testing/trino-tests"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-raptor-legacy"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-accumulo"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-cassandra"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-clickhouse"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-hive",
+        "lib/trino-orc"
+      ]
+    },
+    {
+      "modules": [
+        "plugin/trino-hive",
+        "lib/trino-parquet"
+      ],
+      "options": "-P test-parquet"
+    },
+    {
+      "modules": [
+        "plugin/trino-hive"
+      ],
+      "options": "-P test-failure-recovery"
+    },
+    {
+      "modules": [
+        "plugin/trino-mongodb",
+        "plugin/trino-kafka",
+        "plugin/trino-elasticsearch"
```

Comment:
If we group the 3 together, is it that none of them is run, or all are run?

Reply:
There'll be a single job running any or all of them, depending on which ones were modified. There's a 2-4 minute overhead for compiling stuff.